src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -408,7 +408,7 @@
                 class="actions mr-4"
               >
                 <span
-                  v-if="!hideChangeButtonForSoleProps"
+                  v-if="showChangeButtonForSoleProps"
```

Comment:
You'll see in the store and unit tests that I flipped the meaning of this property.